Kitodo-DataAccess/src/test/java/org/kitodo/dataaccess/AmbiguousDataExceptionTest.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.dataaccess;
+
+import org.junit.Test;
+import org.kitodo.dataaccess.AmbiguousDataException;
+
+public class AmbiguousDataExceptionTest {
+
+    @Test(expected = AmbiguousDataException.class)
+    public void testAmbiguousDataExceptionCanBeThrown() throws AmbiguousDataException {
+        throw new AmbiguousDataException();
```

Comment:
What this kind of test? You are testing the test framework!

Reply:
How else then to test an exception? An exception’s only use is to be thrown, isn’t it?

Reply:
Actually it is not necessary to test it. We don't need 100% code coverage. Our tests should at first be useful.